### PR TITLE
chore: apply v2 trove classifier on v2 python packages

### PR DESCRIPTION
**Pull request by RomainMuller**
Those are currently incorrectly listed against version 1 of the
Framework :: AWS CDK trove classifier. This changes so the v2 branch
correctly uses version 2 here.

----

### All Submissions:

* [X] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/master/INTEGRATION_TESTS.md)?
	* [ ] Did you use `cdk-integ` to deploy the infrastructure and generate the snapshot (i.e. `cdk-integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by RomainMuller** [comment]
The trove classifier registration was requested in https://github.com/pypa/trove-classifiers/pull/98. We do not need to wait for this to be accepted before this can be merged (PyPI will not reject unregistered trove classifiers unless they start with `Private ::`).

**Review by yuth — APPROVED** [review]

**Pull request merged**
